cvat-ui/react_nginx.conf

```diff
@@ -21,6 +21,9 @@ server {
         add_header Cross-Origin-Embedder-Policy "credentialless";
         add_header Expires 0;
         add_header X-Frame-Options "deny";
+        add_header Referrer-Policy "strict-origin-when-cross-origin";
+        add_header X-Content-Type-Options "nosniff";
+        add_header Content-Security-Policy "default-src 'self'; script-src 'self' 'unsafe-inline' https://cdnjs.cloudflare.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; img-src 'self' data: https://app.cvat.ai; font-src 'self' https://fonts.gstatic.com; frame-src 'none'; object-src 'none'; base-uri 'self'; form-action 'self'; frame-ancestors 'none'; manifest-src 'self'; worker-src 'self'; 
```

Comment:
`script-src 'unsafe-inline'` nullifies most of the advantages of CSP - are you sure it is needed?

`https://app.cvat.ai` should not be in there - it's specific to our CVAT instance, and is already covered by `'self'`.